playground/backend/README.md

```diff
@@ -94,6 +94,7 @@ default value and there is no need to set them up to launch locally:
 - `SDK_CONFIG` - is the sdk configuration file path, e.g. default example for corresponding sdk. It will be saved to cloud datastore during application startup (default value = `../sdks.yaml`)
 - `DATASTORE_EMULATOR_HOST` - is the datastore emulator address. If it is given in the environment, the application will connect to the datastore emulator.
 - `PROPERTY_PATH` - is the application properties path (default value = `.`)
+- `CACHE_REQUEST_TIMEOUT` - is the timeout to request data from cache (default value = `30 sec`)
```

Comment:
It should not be longer than HTTP timeout, we need to check these values on startup

Reply:
Make sense, but there is no timeout for requests to retrieve data about precompiled objects. We have timeout only for code processing when a pipeline is created